test/conftest.py

```diff
@@ -146,13 +146,16 @@
         "system": {"path": {'Windows': "C:/tools/cygwin/bin"}},
     },
     'bazel': {
-        "default": "6.3.2",
-        "6.3.2": {"path": {'Linux': '/usr/share/bazel-6.3.2/bin',
-                           'Windows': 'C:/tools/bazel/6.3.2',
-                           'Darwin': '/Users/runner/Applications/bazel/6.3.2'}},
-        "7.1.2": {"path": {'Linux': '/usr/share/bazel-7.1.2/bin',
-                           'Windows': 'C:/tools/bazel/7.1.2',
-                           'Darwin': '/Users/runner/Applications/bazel/7.1.2'}},
+        "default": "7",
+        "6.5.0": {"path": {'Linux': '/usr/share/bazel-6.5.0/bin',
+                           'Windows': 'C:/tools/bazel/6.5.0',
+                           'Darwin': '/Users/runner/Applications/bazel/6.5.0'}},
+        "7.4.1": {"path": {'Linux': '/usr/share/bazel-7.4.1/bin',
+                           'Windows': 'C:/tools/bazel/7.4.1',
+                           'Darwin': '/Users/runner/Applications/bazel/7.4.1'}},
+        "8.0.0": {"path": {'Linux': '/usr/share/bazel-8.0.0/bin',
+                           'Windows': 'C:/tools/bazel/8.0.0',
+                           'Darwin': '/Users/runner/Applications/bazel/8.0.0'}},
```

Comment:
If we put just the major we don't have to update the tests in case we change versions
```suggestion
        "default": "7",
        "6": {"path": {'Linux': '/usr/share/bazel-6.5.0/bin',
                           'Windows': 'C:/tools/bazel/6.5.0',
                           'Darwin': '/Users/runner/Applications/bazel/6.5.0'}},
        "7": {"path": {'Linux': '/usr/share/bazel-7.4.1/bin',
                           'Windows': 'C:/tools/bazel/7.4.1',
                           'Darwin': '/Users/runner/Applications/bazel/7.4.1'}},
        "8": {"path": {'Linux': '/usr/share/bazel-8.0.0/bin',
                           'Windows': 'C:/tools/bazel/8.0.0',
                           'Darwin': '/Users/runner/Applications/bazel/8.0.0'}},
```